AirCasting/Services/DownloadMeasurmentsService.swift

```diff
@@ -118,6 +127,7 @@ final class DownloadMeasurementsService: MeasurementUpdatingService {
                     try UpdateSessionParamsService().updateSessionsParams(session: session, output: output)
```

Comment:
In this case I think we can place a `defer { completion() }` right after `do {` and remove all other completion calls

Reply:
How about those in `catch` blocks?

Reply:
Okay, I know already ;)